src/server/engine_shard_set.cc

```diff
@@ -30,6 +32,17 @@ ABSL_FLAG(bool, cache_mode, false,
           "If true, the backend behaves like a cache, "
           "by evicting entries when getting close to maxmemory limit");
 
+// memory defragmented related flags
+ABSL_FLAG(float, mem_defrag_threshold, 20.0,
```

Comment:
I still think this should be `0.05` instead of `20` and use multiple at line 82 instead of divide.

Reply:
OK

Reply:
it was a leftover from the time this was a uint64_t